matchms/Spectrum.py

```diff
@@ -187,17 +187,31 @@ def set(self, key: str, value):
         self._metadata.set(key, value)
         return self
 
-    def to_dict(self) -> dict:
-        """Return a dictionary representation of a spectrum."""
+    def to_dict(self, export_style: str = "matchms") -> dict:
```

Comment:
This is also used in places like scores, for exporting spectra. This works the same now, since the default is set to "matchms". However, we might want to add the export_style function also to Scores.to_dict()

Reply:
Good point. And should be easy to do.

Reply:
I added a test for the `to_dict()` method. But the export style change is more complex than I thought. I believe, because of the automatically triggered harmonization when using Scores... anyway that will be a new issue.